app/main.js

```diff
@@ -122,10 +148,17 @@ function shutdownDaemon(evenIfNotStartedByApp = false) {
       console.log('Killed lbrynet-daemon process');
     });
   } else if (evenIfNotStartedByApp) {
-    console.log('Killing lbrynet-daemon, even though app did not start it');
-    client.request('daemon_stop', []);
-    // TODO: If the daemon errors or times out when we make this request, find
-    // the process and force quit it.
+    console.log('Stopping lbrynet-daemon, even though app did not start it');
+    client.request('daemon_stop', [], (err, res) => {
+      if (err) {
+        // We could get an error because the daemon is already stopped (good)
```

Comment:
You sure about that last part? I'm pretty sure that if the daemon crashes, the app will close. See https://github.com/lbryio/lbry-app/blob/master/app/main.js#L46